packages/styles/scss/patterns/sub-patterns/card-link/index.scss

```diff
@@ -17,6 +18,7 @@
     text-decoration: none;
     display: flex;
     flex-direction: column;
+    transition: $duration--moderate-01 motion(standard, productive);
```

Comment:
Sorry Anna, but I think `bx--card-link` already has a transition — infact it has two of them thanks to `bx--link` — but get the transition to work on the image I think you need to wrap this with the image selector similar to how you did it below.

```scss
.#{$prefix}--card-link {
  .#{$prefix}--image__img {
    transition: $duration--moderate-01 motion(standard, productive);
  }
}
```

> Does the image component not let us pass in a custom `className` so we could use `bx--card-link__img`? It would help reduce the specificity a little.

Reply:
Oh the image component does let us pass a classname, let me do that and wrap the transition like you mentioned above